supreme_court_predictions/models/xg_boost.py

```diff
@@ -96,7 +96,7 @@ def create(self, df):
             bag_of_words_x,
             bag_of_words_y,
             test_size=self.test_size,
-            random_state=123,
+            random_state=SEED_CONSTANT,
```

Comment:
Done @michplunkett ;) I had missed that tiny bit

Reply:
Ahhhhh yeah!